python/example_code/dynamodb/README.md

```diff
@@ -74,12 +67,139 @@ predictable performance with seamless scalability.*
+- pytest 5.3.5 or later (to run unit tests)
+
+You can install these prerequisites by running the following command in a
+virtual environment:
+
+```
+python -m pip install -r requirements.txt
+``` 
+
+### Getting started with tables, items, and queries
+
+This scenario shows you how to create a DynamoDB table for storing movie data. 
+The scenario loads movies into the table from a JSON-formatted file, walks you 
+through an interactive demo to add, update, and delete movies one at a time, and 
+shows you how to query for sets of movies with various parameters.
+
+The demo uses the Requests package to download a `moviedata.json` file of movie data.
+If you prefer, you can download and extract the file to the `GettingStarted` folder
+from the 
+[Amazon DynamoDB developer guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/samples/moviedata.zip). 
+
+To start the scenario, navigate to the `GettingStarted` folder and run the following 
+at a command prompt:
+
+```
+python scenario_getting_started_movies.py
+```
+
+### Get, write, and delete batches of items
+
+This scenario shows you how to write and retrieve DynamoDB data using batch functions.
+
+Boto3 features a 
+[batch_writer](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/dynamodb.html#batch-writing) 
+function that handles all of the necessary intricacies
+of the Amazon DynamoDB batch API on your behalf. This includes buffering, removing
+duplicates, and retrying unprocessed items.
+
+The demo requires a `moviedata.json` file of movie data in the `batching` folder.
+You can download and extract the file from the 
+[Amazon DynamoDB developer guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/samples/moviedata.zip). 
+
+To start the scenario, navigate to the `batching` folder and run the following at a 
+command prompt:
+
+```
+python dynamo_batching.py
+```  
+
+### Accelerate reads with DAX
+
+Shows you how to use the Amazon DAX Client for Python to read items from a DynamoDB 
+table. Retrieval, query, and scan speeds are measured for both Boto3 and DAX clients 
+to show some of the performance advantages of using DAX.
+
+DAX is a DynamoDB-compatible caching service that provides fast in-memory performance 
+and high availability for applications that demand microsecond latency. For more
+information, see [In-Memory Acceleration with DynamoDB Accelerator (DAX)](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DAX.html). 
+
+#### Running on your local computer
+
+When run on your local computer, only the Boto3 client works.
+
+Each file can be run separately at a command prompt. For example, create the
+table by running the following from a command prompt window.
+
+```commandline
+python 01-create-table.py
+```  
+
+#### Running on a DAX cluster
+
+To run the scripts with the DAX client, you must run them on an Amazon EC2 instance 
+within your VPC, as described in the Python sample application tutorial in the  
```

Comment:
To run the scripts with the DAX client, you must run them on an Amazon Elastic Compute Cloud (Amazon EC2) instance within your virtual private cloud (VPC). This process is described in the Python sample application tutorial in the

Reply:
done